contrib/drivers/clickhouse/clickhouse.go

```diff
@@ -37,7 +38,9 @@ var (
 	errUnsupportedReplace      = errors.New("unsupported method:Replace")
 	errUnsupportedBegin        = errors.New("unsupported method:Begin")
 	errUnsupportedTransaction  = errors.New("unsupported method:Transaction")
-	errSQLNull                 = errors.New("SQL cannot be null")
+	errUpdateNotCondition      = errors.New("there should be WHERE condition statement for UPDATE operation")
+	errDeleteNotCondition      = errors.New("there should be WHERE condition statement for DELETE operation")
+	errUpdateNotAssignment     = errors.New("there should be WHERE condition statement for Assignment operation")
```

Comment:
These errors already handled by package gdb, it is not necessary doing the same checks.